### DISCUSSION TITLE
Add action secret does not list valid characters up front

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#11941] [Actions] [answered]
Here's a pretty form:
<img width="688" alt="image" src="https://user-images.githubusercontent.com/2119212/155216011-36a6a359-04ef-4e04-88e2-e86d0762874c.png">

Note that it provides absolutely no indication about acceptable inputs.
Further note that it provides a multiline text field.

Enter a name including a hyphen (`hello-world`), enter a (long complicated) value into the value field, and click `Add secret`.

> Failed to add secret. Secret names can only contain alphanumeric characters ([a-z], [A-Z], [0-9]) or underscores (\_). Spaces are not allowed. Must start with a letter ([a-z], [A-Z]) or underscores (\_).

<img width="477" alt="image" src="https://user-images.githubusercontent.com/2119212/155217270-c5126b76-12a0-448c-b1fd-d9bdad054660.png">

This is user hostile.

There is no link to the documentation from this page.

It would be incredibly helpful if either just listed the acceptable characters up front, or linked to:
https://docs.github.com/en/actions/security-guides/encrypted-secrets#naming-your-secrets

**Comment by ethomson** [comment, marked as answer]
Thanks for the feedback @jsoref - this looks like a thing that we can improve!

**Comment by Riski11234** [comment]
Failed to add secret. Secret names can only contain alphanumeric characters ([a-z], [A-Z], [0-9]) or underscores (_). Spaces are not allowed. Must start with a letter ([a-z], [A-Z]) or underscores (_).

**Reply by vinyar-launchcg** [reply]
2022 and now it's 2024. Still blows up with little explanation. Why not link to a doc as initial poster suggested.
